docs/src/pages/Conformance.md

```diff
@@ -21,6 +21,41 @@ The HL7 FHIR specification is more than just a data format. It defines an [HTTP
 
 The IBM FHIR Server implements a linear versioning scheme for resources and fully implements the `vread` and `history` interactions, as well as version-aware updates.
 
+### Extended operations
+The HL7 FHIR specification also defines a mechanism for extending the base API with [extended operations](https://www.hl7.org/fhir/R4/operations.html).
+The IBM FHIR Server implements a handful of extended operations and provides extension points for users to extend the server with their own.
+
+Operations can invoked via HTTP POST by passing a Parameters in the body or, for simple operations, via HTTP GET by passing the parameters in the URL.
+
+#### System operations
+System operations are invoked at `[base]/$[operation]`
+
+|Operation|Definition|Short Description|Notes|
+|---------|----------|-----------------|-----|
+| $convert | https://hl7.org/fhir/R4/resource-operation-convert.html | Takes a resource in one form and returns it in another | Converts between JSON and XML but *not* between FHIR versions |
+| $export | https://hl7.org/fhir/uv/bulkdata/STU1/OperationDefinition-export.html | Export data from a FHIR server whether or not it is associated with a patient ||
+| $healthcheck | https://github.com/IBM/FHIR/blob/master/fhir-operation-healthcheck/src/main/resources/healthcheck.json | Check the health of the server | Checks for a valid connection to the database |
```

Comment:
as this is posting to our site, should add (an as of version column?) since the site is agnostic of versions.

Reply:
I feel like this table is already going to be too wide for the site...

Reply:
and I left out `$import` since its not fully baked yet

Reply:
I wonder if we need a better approach for versioning the site in general...

Reply:
I'd suggest linking the operation rather than putting the link following. 
*versioning is a fair point* we'd have to think that through.

Reply:
originally i had a column for whether its spec-defined or custom, but I took that out since it was sort of redundant with the url definition (hl7.org vs github.com).  still, i think you're right about turning those URLs into link targets to save on space...will do that now

Reply:
done